docs/site/guides/data-access/transactions.md

```diff
@@ -7,7 +7,8 @@ keywords:
   Transaction
 tags:
 sidebar: lb4_sidebar
-permalink: /doc/en/lb4/Using-database-transactions.html
+permalink: /doc/en/lb4/data-access-guides-transactions.html
```

Comment:
Please capitalize the first letter: `Data-access-...`

Reply:
See https://github.com/strongloop/loopback-next/pull/5775#discussion_r443347719

I would like to transition our URL scheme to all-lowercase names and eventually to nested paths (`/reference/reserved-binding-keys.html`) to match the directory layout in `docs/site`.

We are already following the new style in migration guide, see e.g. here:

https://github.com/strongloop/loopback-next/blob/ae6427322451c914ae54f44dbb656981e7fbbb81/docs/site/migration/datasources.md#L7

Let's discuss in #5798